flutter_bird_app/lib/main.dart

```diff
@@ -1,30 +1,51 @@
+import 'package:flutter/foundation.dart' show kIsWeb;
 import 'package:flutter/material.dart';
 import 'package:flutter_bird/controller/flutter_bird_controller.dart';
 import 'package:flutter_bird/view/main_menu_view.dart';
 import 'package:provider/provider.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:flutter_line_liff/flutter_line_liff.dart';
 
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+   if (const String.fromEnvironment('FLUTTER_ENV') != 'production') {
+    await dotenv.load(fileName: ".env");
+  } 
 
-void main() {
-  dotenv.load(fileName: ".env");
-  runApp(const MyApp());
+  print("Starting application...");
+  const String liffId = String.fromEnvironment('LIFF_ID');
```

Comment:
Where is this used?

Reply:
This variable was used previously, but is no longer needed. Delete. Thanks!